introduction.md

```diff
@@ -3,3 +3,13 @@
 Hi, my name is Daniel Santana, but they call me danielsan.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?